packages/forms/resources/views/components/section.blade.php

```diff
@@ -16,20 +16,31 @@
     ][$formComponent->getColumnSpan()]
 @endphp
 
-<div class="{{ $columnSpanClass }} space-y-3">
-    @if ($formComponent->getHeading())
+<section x-data="{ open: true }" aria-labelledby="section-heading-{{ $formComponent->getId() }}" class="{{ $columnSpanClass }} space-y-4 p-4 rounded border border-gray-200 bg-gray-50">
+    <div class="flex items-start justify-between space-x-4">
         <div class="space-y-1">
-            <h3 class="text-lg leading-tight font-medium">
+        @if ($formComponent->getHeading())
+            <h2 id="section-heading-{{ $formComponent->getId() }}" class="text-lg font-medium leading-tight">
                 {{ __($formComponent->getHeading()) }}
-            </h3>
+            </h2>
+        @endif
 
-            @if ($formComponent->getSubheading())
-                <p class="text-gray-700 text-sm">
-                    {{ __($formComponent->getSubheading()) }}
-                </p>
-            @endif
+        @if ($formComponent->getSubheading())
+            <p class="text-sm text-gray-500">
+                {{ __($formComponent->getSubheading()) }}
+            </p>
+        @endif
         </div>
-    @endif
-
-    <x-forms::layout :schema="$formComponent->getSchema()" :columns="$formComponent->getColumns()" />
-</div>
+        
+        <div class="flex">
+            <button aria-controls="section-content-{{ $formComponent->getId() }}" @click.prevent="open = !open" class="flex p-2 -m-2 text-gray-400 transition-colors duration-200 hover:text-gray-700">
```

Comment:
Think this should be wrapped in an "if collapsible" check.

Reply:
Done! Just pushed an update to allow the following:

```php
// collapsible false by default when creating a section to maintain current functionality and not hide things inadvertently
Components\Section::make($heading, $subheading, $schema = [], $collapsible = false)

Components\Section::make()->collapsible() // fluent method for enabling collapsible functionality (I need some help withe how to create a callback on this method to allow for logic to set true|false

Components\Section::make()
    ->collapsed() // sets the initial state of the section to collapsed, and enables 'collapsible' to ensure it can be toggled back open again
```